application/modules/guest/views/index.php

```diff
@@ -3,32 +3,36 @@
 </div>
 
 <div id="content">
-
+    <?php if (!$this->session->userdata('user_all_clients')) { ?>
```

Comment:
Is that check correct here? Why did you add it?

Reply:
Yeah that's right. it's an improvement that I already published and it's embedded in the system. The users who can see all the clients (who are basically the accountants), do not visualize the budgets since they only need the invoices and the payments.